schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
[OPTIONAL] you can  mark the id column as primary key
`PRIMARY KEY (id)`

Reply:
Great idea! 
![image](https://user-images.githubusercontent.com/105387310/205913013-279a4d0c-57f3-48e9-b035-8137e06a2c55.png)
Done